src/ansys/grantami/recordlists/_connection.py

```diff
@@ -639,13 +639,33 @@ def get_resolvable_items(self, all_items: List[RecordListItem]) -> List[RecordLi
             ]
         return resolvable_items
 
-    def _get_db_map(self) -> Dict[str, str]:
+    def _get_db_map(self) -> Dict[str, List[str]]:
         dbs = self._db_schema_api.get_all_databases()
-        return {db.guid: db.key for db in dbs.databases}
+        db_map: Dict[str, List[str]] = {}
+        for db in dbs.databases:
+            db_keys = db_map.setdefault(db.guid, list())
```

Comment:
Is this better than using a defaultdict?

Reply:
I did wonder if type hinting was the cause. You can use `defaultdict` as a type directly and it should work, and it results in one fewer lines of code. But I don't feel particularly strongly about it.